modules/meta.analysis/DESCRIPTION

```diff
@@ -9,10 +9,8 @@ Authors@R: c(person("Mike","Dietze"),
              person("Dan"," Wang"),
              person("Carl", "Davidson"),
              person("Rob","Kooper"),
-             person("Shawn", "Serbin"))
-Author: David LeBauer, Mike Dietze, Xiaohui Feng, Dan Wang, Carl
-    Davidson, Rob Kooper, Shawn Serbin
-Maintainer: David LeBauer <dlebauer@illinois.edu>
```

Comment:
I guess we didn't have the maintainer recorded in `Authors@R` after all. GitHub won't let me add it as a suggestion because it's too far from an already-changed line, but this can be fixed by editing line 7  to read `person("David", "LeBauer", role = c("aut", "cre"), email = "dlebauer@email.arizona.edu")`